mobile/examples/object_detection/ios/ORTObjectDetection/download.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Maybe this file can be renamed to prepare_model.sh?
Also do you want to delete the tflite model after the coversion?